### PR TITLE
175 update compression docs

### PR DESCRIPTION
**Pull request by daveatweaviate**
[issue](https://github.com/weaviate/weaviate-edu-team-tasks/issues/175)
[staging](https://175-update-compression-docs--tangerine-buttercream-20c32f.netlify.app/developers/weaviate/configuration/indexes)

### What's being changed:
Refator of the indexes page. Adds new details.

### Type of change:
- [x] **Documentation** updates (non-breaking change to fix/update documentation)
### How Has This Been Tested?

- [x] **Github action** – automated build completed without errors
- [x] **Local build** - the site works as expected when running `yarn start`

### CODE REVIEW

File: blog/2023-02-21-combining-langchain-and-weaviate/index.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ Large Language Models (LLMs) have revolutionized the way we interact and communi
 This blog post will begin by explaining some of the key concepts introduced in LangChain and end with a demo. The demo will show you how to combine LangChain and Weaviate to build a custom LLM chatbot powered with semantic search!
 
 ## Sequential Chains
-[Chains](https://docs.langchain.com/docs/components/chains/) enable us to combine multiple LLM inferences together. As you can guess from the name, sequential chains execute their links in a sequential order. It takes one input/output and then uses the output for the next step.
+[Chains](https://python.langchain.com/docs/modules/chains/) enable us to combine multiple LLM inferences together. As you can guess from the name, sequential chains execute their links in a sequential order. It takes one input/output and then uses the output for the next step.
```
Comment (daveatweaviate) [inline blog/2023-02-21-combining-langchain-and-weaviate/index.mdx:21]: Unrelated fix - langchain has updated it's API and the old page 404s now

**Review by daveatweaviate — COMMENTED** [review]
Notes for  reviewers

**Comment by daveatweaviate** [comment]
@databyjp would you have a look at this page when you get a moment? 

It moves a lot of body text into tables.
Moves some text into new sections.
Adds new definitions and explanations.
Adds a new example for dynamic ef

**Comment by databyjp** [comment]
Hi @daveatweaviate, I've reviewed the doc & made a few additional changes. Pls feel free to lmk if you want to discuss - and also feel free to merge when convenient. :)

**Comment by daveatweaviate** [comment]
@databyjp thanks for the suggestions - would you make them as comments next time rather than modifying the PR? That makes a mess of my branch and causes all sorts of merge conflicts. It also makes it harder to figure out what the changes are.

**Review by daveatweaviate — COMMENTED** [review]
@databyjp  Thanks for your feedback. I think I've gotten back all the edits after the merge problems. This is ready for another pass if you want to take a look. If not, I can merge it in.
